examples/snippets/utils/client/manual_auth.py

```diff
@@ -1,26 +1,22 @@
 """
-this snippet shows how a tamr client object can be created
-with manual passing of credentials during runtime.
+Snippet for creating a tamr client object with manual passing of credentials during runtime.
 """
 from tamr_toolbox import utils as tbu
-from tamr_toolbox.utils.client import Client
 from getpass import getpass
 
 
-def manual_auth(config: dict) -> Client:
-    """
-    This function allows for setting up credentials in terminal when creating a unify client object
-    - using the getpass function will conceal password phrase and requires manual human user input
-    - this approach cannot be coupled with fully automated workflows (e.g., crobtab jobs)
-    """
+## a. using the getpass function will conceal password phrase and requires manual human user input
+## b. this approach cannot be coupled with fully automated workflows (e.g., crobtab jobs)
+## c. it is presumed that when setting AUTH manually the config.yaml file does not contain the
+## username and password fields
```

Comment:
This info might be more appropriate alongside the config loading (see the suggestion below)